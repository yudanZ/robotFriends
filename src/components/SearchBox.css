.input_container {
    margin: 20px;
    width: 50%;
    text-align: center;
}

.input_container input{
    margin-left: 50%;
    text-align: center;
    height: 4rem;
}