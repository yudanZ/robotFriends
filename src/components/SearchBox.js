import React from 'react';
import './SearchBox.css';

const SearchBox = ({ searchfield, searchChange}) => {
    return (
        <div className='input_container'>
            <label htmlFor='search-input' className='serach-label'>Search Robots: </label>
            <input
                id='search-input'
                arial-label='Search Robots' 
                className='form-control' 
                type='search' 
                placeholder='search robots'
                onChange = {searchChange} 
            />
        
        </div>
        
    );
}

export default SearchBox;