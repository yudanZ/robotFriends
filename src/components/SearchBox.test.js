import { shallow } from 'enzyme';
import React from 'react';
import SearchBox from './SearchBox';

it('expect to render <SearchBox /> component', () => {
    expect(shallow(<SearchBox />)).toMatchSnapshot();
})