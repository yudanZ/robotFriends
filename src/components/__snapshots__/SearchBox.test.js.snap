// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render <SearchBox /> component 1`] = `
<div
  className="input_container"
>
  <label
    className="serach-label"
    htmlFor="search-input"
  >
    Search Robots: 
  </label>
  <input
    arial-label="Search Robots"
    className="form-control"
    id="search-input"
    placeholder="search robots"
    type="search"
  />
</div>
`;
